Calculate.sketchplugin/Contents/Sketch/script.js

```diff
@@ -85,11 +106,17 @@ let showError = function (layer, e) {
   return;
 }
 
+var changedName = function (context) {
+  console.log("Changed name!");
+  calculate();
+}
+
 var changedText = function (context) {
```

Comment:
Since `context` is an Sketch global variable, I think we shouldn't requite it as an argument.

Reply:
Context is not global. It is passed when the event happens, and should contain the details of the operation that triggered it.

Reply:
Anyway I'm still talking with Ale about this, as it doesn't seem to be working properly.

Reply:
I think it is, at least they say it here: <img width="659" alt="screenshot 2019-02-21 at 10 08 35" src="https://user-images.githubusercontent.com/4933/53156950-c9f85980-35c0-11e9-9c49-5025f0e58551.png"> (and also the scripts works fine if you remove it from the params of the method).

Reply:
Ok, let me check it later.